nautobot/core/tests/test_views_utils.py

```diff
@@ -98,3 +99,9 @@ def test_check_filter_for_display(self):
         # with self.assertEqual(
         #     check_filter_for_display(device_filter_set_filters, "manufacturer", ["fake_slug"]), expected_output
         # )
+
+
+class CheckCountRelatedSubquery(TestCase):
```

Comment:
This test isn't failing for me on develop. You need the inventory items with a common manufacturer for the bug to present.